corehq/apps/app_manager/templates/app_manager/partials/build_errors.html

```diff
@@ -317,5 +317,12 @@
                 {% trans "Errors Building an Application" %}
             </a>
         </p>
+        {% if in_iframe %}
+        <script>
+            $('#build-errors a').each(function(index) {
+                $(this).attr('target', '_parent');
+            });
+        </script>
+        {% endif %}
```

Comment:
Hmm...I see what you mean, though I still think eventually we're going to want to find a way around this. Yeah, let's catch up offline.